xclim/core/formatting.py

```diff
@@ -77,7 +81,8 @@ def format(self, format_string: str, /, *args: Any, **kwargs: dict) -> str:
         for k, v in DEFAULT_FORMAT_PARAMS.items():
             if k not in kwargs:
                 kwargs.update({k: v})
-        return super().format(format_string, *args, **kwargs)
+        kwargs["np"] = np  # noqa
```

Comment:
A bit ugly, maybe there is a better way to add numpy in jinja rendering namespace